tests/testthat/test_misc.R

```diff
@@ -8,8 +8,29 @@ test_that("now_utc returns time in UTC", {
   expect_equal(attr(now, "tzone"), "UTC")
 })
 
+test_that("print_list returns expected results", {
+  l1 <- list(x = 1, y = 2)
+  l2 <- list(x = 1, y = "foo")
+  l3 <- list(x = list(y = "foo"), z = "bar")
+  res1 <- print_list(l1)
+  res2 <- print_list(l2)
+  res3 <- print_list(l3)
+  expect_equal(res1, "list(x = 1, y = 2)")
+  expect_equal(res2, "list(x = 1, y = \"foo\")")
+  expect_equal(res3, "list(x = list(y = \"foo\"), z = \"bar\")")
+})
+
+test_that("print_list can remove outer list() call", {
+  l <- list(x = 1, y = 2)
+  expect_equal(print_list(l, wrapper = FALSE), "x = 1, y = 2")
+})
+
```

Comment:
it works totally for for the case, but maybe one of these tests should have a vector? e.g.:

```
print_list(list(formulation = c("a", "b")), wrapper = F)
```